src/JustEat.StatsD/V2/StatsDUtf8Formatter.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Text;
+
+namespace JustEat.StatsD.V2
+{
+    internal class StatsDUtf8Formatter
+    {
+        private readonly byte[] _utf8Prefix;
+
+        public StatsDUtf8Formatter(string prefix = "")
+        {
+            _utf8Prefix = string.IsNullOrWhiteSpace(prefix) ? new byte[0] : Encoding.UTF8.GetBytes(prefix + ".");
```

Comment:
Use `Array.Empty<byte>()`?

Reply:
needs 4.6 at least, we are pointing to 4.5.1

Reply:
Could use it for the newer TFMs if we're going to go down the `#if FRAMEWORK` and trying to reduce allocations though...